tests/integrationv2/common.py

```diff
@@ -7,7 +7,7 @@
 
 
 from constants import TEST_CERT_DIRECTORY
-from global_flags import get_flag, S2N_NO_PQ, S2N_FIPS_MODE
+from global_flags import get_flag, S2N_NO_PQ, S2N_FIPS_MODE, S2N_USE_CRITERION
 
```

Comment:
Why is this import necessary? What uses S2N_USE_CRITERION?

Reply:
it's not needed, removed.